### PR TITLE
US 1583733 Add missing language IDs - 33

### PR DESCRIPTION
**Pull request by v-thpra**
US 1583733 - Fix CATS P1 issues by adding missing language IDs to code blocks.
- Used "bat" for .bat file examples.
- Used "console" for sample program console commands and console output examples.

Contributes to #2192

### CODE REVIEW

File: docs/framework/wcf/samples/membership-and-role-provider.md [hunk 1]
```diff
@@ -188,6 +188,6 @@ The Membership and Role Provider sample demonstrates how a service can use the A
   
      The following lines in the Setup.bat batch file copy the server certificate into the client trusted people store. This step is required because certificates generated by Makecert.exe are not implicitly trusted by the client system. If you already have a certificate that is rooted in a client trusted root certificate—for example, a Microsoft-issued certificate—this step of populating the client certificate store with the server certificate is not required.  
   
-    ```  
+    ```bat
```
Comment (HongGit) [inline docs/framework/wcf/samples/membership-and-role-provider.md]: should this be console?

File: docs/framework/wcf/samples/mtom-encoding.md [hunk 2]
```diff
@@ -64,7 +64,7 @@ Press <ENTER> to terminate client.
   
 1. Install ASP.NET 4.0 using the following command.
```
Comment (HongGit) [inline docs/framework/wcf/samples/mtom-encoding.md]: I have commented on other places as well.  Please update this to something else other than running aspnet_regiis.exe.  Maybe simply say "enable ASP.NET"?

File: docs/framework/wcf/samples/net-tcp-port-sharing-sample.md [hunk 3]
```diff
@@ -121,7 +121,7 @@ Press <ENTER> to terminate client.
   
 1. Install ASP.NET 4.0 using the following command.
```
Comment (HongGit) [inline docs/framework/wcf/samples/net-tcp-port-sharing-sample.md]: see my comments above

File: docs/framework/wcf/samples/one-time-setup-procedure-for-the-wcf-samples.md [hunk 4]
```diff
@@ -25,7 +25,7 @@ The **ServiceModelSamples** virtual directory is used for building and running a
     > [!WARNING]
     > If IIS is reinstalled, the following commands will need to be run again.
 
-    ```
+    ```console
     "%WINDIR%\Microsoft.Net\Framework\v4.0.30319\aspnet_regiis" –i –enable
```
Comment (HongGit) [inline docs/framework/wcf/samples/one-time-setup-procedure-for-the-wcf-samples.md]: my comments above

File: docs/framework/wcf/samples/saml-token-provider.md [hunk 5]
```diff
@@ -341,7 +341,7 @@ This sample demonstrates how to implement a custom client SAML token provider. A
 
      The following lines in the Setup.bat batch file copy the server certificate into the client trusted people store. This step is required because certificates generated by Makecert.exe are not implicitly trusted by the client system. If you already have a certificate that is rooted in a client trusted root certificate—for example, a Microsoft-issued certificate—this step of populating the server certificate store with the issuer certificate is not required.
 
-    ```
+    ```bat
```
Comment (HongGit) [inline docs/framework/wcf/samples/saml-token-provider.md]: console?

**Comment by v-thpra** [comment]
@mairaw @HongGit To resolve the last two comments I need a suggested change. I think that @HongGit is asking that we rewrite the code line or replace it, but I need more information to be able to do that. Thanks!

**Review by mairaw — APPROVED** [review]
Thanks @v-thpra. This task was specifically about adding missing identifiers. If content needs revision (and probably a lot of the WCF content does), this would have to be done in a different PR.

**Pull request merged**